bots/ninja_bot.py

```diff
@@ -272,15 +278,14 @@ def evaluate_ap(self, weight=1):
         return total_ap_value, d
 
     def evaluate_kills(self, weight=1):
-        if not self.uid in self.alive_ids:
-            return float('-inf'), f'LOSING STATE'
-        doomed = len(self.doomed_ids)
-        alive = len(self.enemy_ids) - doomed
-        if alive == 0:
+        if len(self.enemy_ids) == 0:
+            if self.pos in self.doomed_tiles:
+                return 1_000, f'DRAW STATE'
```

Comment:
you really don't like Enums

Reply:
In this particular case, the number is a value and the string is for debug logs. Check out vzero to see some hardcore enum abuse :)